circle.yml

```diff
@@ -0,0 +1,34 @@
+machine:
+  environment:
+    CXX: g++-4.8
+    BABEL_ENV: cover
+    CACHE: $HOME/cache
+    CMAKE_SHORT_VERSION: 3.4
+    CMAKE_VERSION: 3.4.3
+    PATH: $CACHE/cmake-$CMAKE_VERSION/bin:$PATH
+    JASMINE_TIMEOUT: 15000
+  node:
+    version: 5.12
+  post:
+    - pyenv global 2.7.11
+    - if [ "$CIRCLE_NODE_INDEX" -eq 1 ] ; then pyenv global 3.4.4 ; fi
+    - pyenv rehash
+
+dependencies:
+  cache_directories:
+    - "~/cache"
+  override:
+    - pyenv rehash
+    - sudo apt-get update; sudo apt-get install libgif-dev
+    - ./scripts/install_cmake.sh
+    - npm install; npm update; npm prune
```

Comment:
I removed the `install`, but the `update` and `prune` steps are technically necessary to get a clean environment.  Subsequent `npm install` executions only take a few seconds.